src/Google.Api.Gax/Api/Gax/MetadataExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Grpc.Core;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Google.Api.Gax
+{
+    internal static class MetadataExtensions
+    {
+        public static void Add(this Metadata metadata, IEnumerable<Metadata.Entry> items)
```

Comment:
List<T> has a method called `AddRange`, should we use that name? Not that it matters much since it is internal.